docs/monitor/configure-alarms.md

```diff
@@ -75,13 +76,18 @@ your systems, containers, and applications work.
 Read Netdata's [health reference](/health/REFERENCE.md#health-entity-reference) for a full listing of the format,
 syntax, and functionality of health entities.
 
-To write a new health entity, use `edit-config` to create a new file inside of the `health.d/` directory.
+To write a new health entity into a new file, navigate to your [Netdata config directory](/docs/configure/nodes.md),
+then use `touch` to create a new file in the `health.d/` directory. Use `edit-config` to start editing the file.
```

Comment:
@joelhans , please, add a short text to explain for the users that files inside `health.d`must have an extension `.conf`. Thank you!